README.md

```diff
@@ -1 +1,40 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> project on config linters
+
+
+## Built With
+
+- HTML - 5
+- CSS - 3
+
```

Comment:
Hi, @AmitKlohan Good Job including a well-documented ReadMe File. I suggest you also include **Package Manager: Node, NPM** to the built with section of your ReadMe File.